.env.example

```diff
@@ -0,0 +1,10 @@
+MONGO_URI=mongodb+srv://LVien:<password>@practicum.twrhx3o.mongodb.net/databasename?retryWrites=true&w=majority
+("url from mongodb connection")
+
+JWT_SECRET=NcRfUjXn2r5u8x/A%D*G-KaPdSgVkYp3
```

Comment:
Can you not include any secret value here? We want everyone to have to generate their own. Having this here might encourage someone to use this value which would not be preferable. You can just substitute it with "SECRET_VALUE" or something along those lines.

Reply:
Noted. I have made some changes to that file so the secret value will be stated as anonymous.

Reply:
`JWT_SECRET=SECRET_VALUE
("signature or a code that is used to identify the authenticity of the token,if you want to generate a secure key you can go to 
https://allkeysgenerator.com/, choose encryption key then 256bit as security level, the copy and paste the key here")`